assets/js/data/cart/reducers.ts

```diff
@@ -1,26 +1,41 @@
 /**
  * External dependencies
  */
-import { camelCase, mapKeys } from 'lodash';
+import type { CartItem, CartResponseItem, Cart } from '@woocommerce/types';
 
 /**
  * Internal dependencies
  */
 import { ACTION_TYPES as types } from './action-types';
-import { defaultCartState } from '../default-states';
+import { defaultCartState, CartState } from '../default-states';
+import type { ResponseError } from '../types';
+
+type AnyAction = {
+	type: string;
+	error?: ResponseError | null;
+	response?: Cart;
+	couponCode?: string;
+	cartItem?: CartResponseItem | null;
+	cartItemKey?: string;
+	isResolving?: boolean;
+	[ key: string ]: unknown;
```

Comment:
Probably good to remove this otherwise basically any key can be in this type. That way we have to update it if we add additional actions to the store?

Reply:
Removed entirely as a result of work in 263f9e6